subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/resolveengine/graph/builder/DependencyGraphBuilder.java

```diff
@@ -260,6 +260,15 @@ public void execute(BuildOperationQueue<RunnableBuildOperation> buildOperationQu
         }
     }
 
+    private ComponentIdentifier toComponentId(ModuleVersionIdentifier id, Map<ModuleVersionIdentifier, ComponentIdentifier> componentIdentifierCache) {
```

Comment:
:o: method can be static

Reply:
Is there a benefit to making it static?

Reply:
I like it better because it makes it clear it's a helper method, stateless, that could be extracted if needed.

Reply:
Understood